benchmark/benchmark.go

```diff
@@ -193,6 +121,22 @@ func DoUnaryCall(tc testpb.TestServiceClient, reqSize, respSize int) {
 	}
 }
 
+// DoStreamingcall performs a streaming RPC with given stub and request and response size.client side
```

Comment:
this comment is incorrect. It only does 1 round-trip for a single streaming rpc.